FormalConjectures/ErdosProblems/213.lean

```diff
@@ -0,0 +1,38 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.ProblemImports
+
+/-!
+# Erdős Problem 213
+
+*Reference:* [erdosproblems.com/213](https://www.erdosproblems.com/213)
+-/
+
+open scoped EuclideanGeometry
+
+/--
+Let n ≥ 4. Are there n points in ℝ², no three on a line and no four on a circle,
+such that all pairwise distances are integers?
+-/
+@[category research open, AMS 52]
+theorem erdos_213 : ∀ n : ℕ, n ≥ 4 →
```

Comment:
Here I would be tempted to do the following: define a predicate `IsGood` (perhaps with a better name:)) stating that for a given `n`, such a set exists. You would then be able to state the main result as `(∀ n : ℕ, n ≥ 4 →IsGood n) ↔ answer(sorry)`. 

One reason this would be useful is that it would allow you to state some of the proved special cases, which is often a good sanity check that the main conjecture statement is correct.